src/components/About/About.js

```diff
@@ -1,10 +1,95 @@
 import React from 'react';
+import { 
+  Octokit, 
+} from '@octokit/rest';
 import styles from './About.module.css';
+import CircularProgress from '@material-ui/core/CircularProgress';
 
-const About = () => (
-    <div className={styles.wrapper}>
-        <h1>Обо мне</h1>
-    </div>
-);
+const octokit = new Octokit();
+
+class About extends React.Component {
+
+  state = {
+    isLoading: true,
+    repoList: [],
+        isError: false,
+        userInfo: [],
+        username: 'MylnikovaO',
+  };
+
+  componentDidMount() {
+
+    octokit.repos.listForUser({
+      username: this.state.username,
+    })
+
+    .then(({ data }) => {
+      this.setState({
+        repoList: data,
+        isLoading: false,
+      });
+    })
+
+      .catch(() => {
+        this.setState({
+          isError: true,
+          isLoading: false
+        })
+    });
+
+    octokit.users.getByUsername({
+      username: this.state.username
+        }).then((response) => {
+          this.setState({
+            avatarURL: response.data.avatar_url,
+            name: response.data.name
+        });
+    })
+
+          .catch(err => {
+            this.setState({
+              isLoading: false,
+              isError: true,
+            });
+        });
+    }
+
+    render () {
+      const { 
+        isLoading, 
+        repoList, 
+        isError, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5619303)